contracts/IColony.sol

```diff
@@ -420,14 +409,13 @@ contract IColony is IRecovery {
   /// @return rating Rating the user received
   function getTaskRole(uint256 _id, uint8 _role) public view returns (address user, bool rateFail, uint8 rating);
 
-  // Implemented in ColonyFunding.sol
-  /// @notice Return 1 / the fee to pay to the network. e.g. if the fee is 1% (or 0.01), return 100
-  /// @return feeInverse The inverse of the network fee
-  function getFeeInverse() public pure returns (uint256 feeInverse);
+  /// @notice Set the reward inverse to pay out from revenue. e.g. if the fee is 1% (or 0.01), set 100
+  /// @return _rewardInverse The inverse of the reward
```

Comment:
```suggestion
```

Reply:
This should actually be a `@param` natspec not `@return`

Reply:
👍